org.sparkleshare.SparkleShare.json

```diff
@@ -0,0 +1,130 @@
+        {
+            "name": "git",
+            "make-args": ["NO_PERL=1"],
+            "make-install-args": ["NO_PERL=1"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.kernel.org/pub/software/scm/git/git-2.14.1.tar.xz",
+                    "sha256": "6f724c6d0e9e13114ab35db6f67e1b2c1934b641e89366e6a0e37618231f2cc6"
+                }
+            ]
+        },
+        {
+            "name": "git-lfs",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/git-lfs/git-lfs/releases/download/v2.3.0/git-lfs-linux-amd64-2.3.0.tar.gz",
+                    "sha256": "5913ed4d023efe30a3f8f536db7cd97c7b76ba68b189baef8428b1d71d82f2f4"
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir --parents /app/libexec/git-core/",
+                "cp git-lfs /app/libexec/git-core/"
+            ]
+        },
+        {
+            "name": "gtk-sharp3",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mono/gtk-sharp.git",
+                    "commit": "9ef7d3f394b9d485674c59e92db24ac66fd9f16b"
+                }
+            ]
+        },
+        {
+            "name": "dbus-sharp",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/hbons/dbus-sharp",
+                    "commit": "d415624aaa01ec5fbca8f1c65373ba0e21115c69"
+                }
+            ]
+        },
+        {
+            "name": "dbus-sharp-glib",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/hbons/dbus-sharp-glib",
+                    "commit": "23f6dba3764a3bb229a8538e8149a91a6a7fd319"
+                }
+            ]
+        },
+        {
+            "name": "notify-sharp",
+            "config-opts": ["--disable-docs"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/hbons/notify-sharp",
+                    "commit": "1f56c37349b7750f3fb4cb6ac88ccbaa62c15d30"
+                }
+            ] 
+        },
+        {
+            "name": "webkit2-sharp",
+            "no-parallel-make": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/hbons/webkit2-sharp",
+                    "commit": "a59fd76dd730432c76b12ee6347ea66567107ab9"
+                }
+            ] 
+        },
+        {
+            "name": "sparkleshare",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/hbons/SparkleShare",
+                    "commit": "5ae6a8252d1d317abc38169e6a168403934b5a98"
```

Comment:
Looks like you have no release tags upstream, do you plan on making tags? Flathub is not intended for random git builds but rather stable versioned releases.

Reply:
Using a tag now.